src/test/java/harness/TestUser.java

```diff
@@ -108,7 +99,7 @@ public GoogleCredentials getCredentialsWithCloudPlatformScope() throws IOExcepti
   /** Get domain-wide delegated Google credentials for this user. */
   private GoogleCredentials getCredentials(List<String> scopes) throws IOException {
     // get a credential for the test-user SA
-    Path jsonKey = Path.of("rendered", "test-user-account.json");
+    Path jsonKey = Path.of("rendered", TestConfig.getTestConfigName(), "test-user-account.json");
```

Comment:
Instead of hard-coding the SA key file names, I think it would be cleaner to make these properties of the test config. The filenames we currently expect `render-config.sh` to generate are not really documented anywhere, and this would obviate the need for that.
e.g. in json `"domainWideDelegationSA": "rendered/broad/test-user-account.json")`
       in java `TestConfig.getDomainWideDelegationSAFile()`

Reply:
Discussed offline. Personally I prefer having fewer knobs.

This discussion brings up a good point: I need to document "Here's what to do for a new deployment". I will do that as part of PF-1289. The documentation will already say "Set up a `render-config.sh` for your deployment`". (And it's fairly certain someone will be reading this documentation as part of setting up a new deployment :)) So the documentation can just say "Create files name such-and-such".